app/code/Magento/Theme/Model/ResourceModel/Theme/Grid/Collection.php

```diff
@@ -5,11 +5,62 @@
  */
 namespace Magento\Theme\Model\ResourceModel\Theme\Grid;
 
+use Magento\Theme\Model\ResourceModel\Theme\Collection as ThemeCollection;
+use Magento\Framework\Api\Search\SearchResultInterface;
+use Magento\Framework\Api\Search\AggregationInterface;
+
 /**
  * Theme grid collection
  */
-class Collection extends \Magento\Theme\Model\ResourceModel\Theme\Collection
+class Collection extends ThemeCollection implements SearchResultInterface
```

Comment:
Hi @simpleadm, please make some refactoring:
1) The collection shouldn't extend the ThemeCollection collection, it should only implement SearchResultInterface.
2) Use ThemeCollection to getting access to data inside implemented methods.